app/views/foreman/job_templates/install_errata.erb

```diff
@@ -6,10 +6,10 @@ description_format: 'Install errata %{errata}'
 feature: katello_errata_install
 provider_type: SSH
 template_inputs:
-- name: errata
-  description: A comma separated list of errata to install
+- name: Errata Search Query
```

Comment:
```suggestion
- name: Errata search query
```

On all the new pages we've been instructed to use this style of capitalization.

Reply:
I think this broke the display of the job name. All my tasks now say
```
Install errata ''
```
but they should say the errata ID:
```
Install errata '2021-blahblah-30148u'
```
I think if you change line 5 above to
```
description_format: 'Install errata %{Errata search query}'
```
it will maybe hopefully fix it.

Reply:
fixed